tests/environment_test.go

```diff
@@ -14,13 +15,12 @@ import (
 )
 
 func TestMultipleEnvironments(t *testing.T) {
-	m := it.MysqlDatabaseT(t)
-	m.ImportIcingaDbSchema()
+	rdb := getDatabase(t)
```

Comment:
rdb implies Redis!

Reply:
What?

Reply:
If you grep in the icingadb repo... OK, forget it.